pkg/ensurance/executor/mem_usage.go

```diff
@@ -0,0 +1,70 @@
+package executor
+
+import (
+	"sync"
+
+	"k8s.io/klog/v2"
+
+	"github.com/gocrane/crane/pkg/ensurance/executor/podinfo"
+	"github.com/gocrane/crane/pkg/ensurance/executor/sort"
+	"github.com/gocrane/crane/pkg/metrics"
+	"github.com/gocrane/crane/pkg/utils"
+)
+
+func init() {
+	registerMetricMap(memUsage)
+}
+
+var memUsage = metric{
+	Name:           MemUsage,
+	ActionPriority: 5,
+	Sortable:       true,
+	SortFunc:       sort.MemUsageSort,
+
+	Throttleable:       false,
+	ThrottleQuantified: false,
+	ThrottleFunc:       nil,
+	RestoreFunc:        nil,
+
+	Evictable:       true,
+	EvictQuantified: true,
+	EvictFunc:       memUsageEvictPod,
+}
+
+func memUsageEvictPod(wg *sync.WaitGroup, ctx *ExecuteContext, index int, totalReleasedResource *ReleaseResource, EvictPods EvictPods) (errPodKeys []string, released ReleaseResource) {
+	wg.Add(1)
+
+	// Calculate release resources
+	released = releaseMemUsage(EvictPods[index])
+	totalReleasedResource.Add(released)
```

Comment:
If the pod evicted failed, need we subtract  the release?

Reply:
Ok, i will make a new pr to fix this.